src/shapes.tsx

```diff
@@ -31,7 +31,7 @@ export const SHAPES = [
       </svg>
     ),
     value: "diamond",
-    key: "d",
+    key: "g",
```

Comment:
Is there a reason why "g", it doesn't anywhere in "diamond"

Reply:
no particular reasons, just picked free letter in the left part of keyboard

Reply:
I see other options are N and M possibly. They're both on the right side of keyboard , but relatively close to left still.